app/src/components/TruncateText.tsx

```diff
@@ -14,14 +14,18 @@ export const TruncateText = (props: TruncateTextProps): React.ReactElement => {
     setShouldTruncateText(!shouldTruncateText);
   };
   return (
-    <Stack direction={Direction.Vertical} isFullWidth={true} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
+    <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
```

Comment:
are you 100% sure this works for longer text too? can you put some more screenshot examples pls if it does